src/components/cats/index.js

```diff
@@ -8,6 +9,11 @@ export default props => (
 
     <p>{props.currentCat.description}</p>
 
+    <img src={[adoptedCats]} />
```

Comment:
This will be displaying multiple cats by the time we are done, so you will have to display multiple images.